<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmDialog</class>
 <widget class="QDialog" name="AlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlarmDialog</string>
  </property>
<widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
    <rect>
        <x>0</x>
        <y>5</y>
        <width>450</width>
        <height>500</height>
    </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
            <spacer name="horizontalSpacer">
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
            <size>
                <width>20</width>
                <height>40</height>
            </size>
            </property>
            </spacer>
            </item>
        <item>
    <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
        <rect>
        <x>0</x>
        <y>0</y>
        <width>401</width>
        <height>481</height>
        </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
        <item>
        <widget class="QLabel" name="label">
        <property name="text">
        <string>Adresse</string>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <widget class="QLineEdit" name="lineEdit">
        <property name="text">
        <string/>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <spacer name="verticalSpacer">
        <property name="orientation">
        <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
        <size>
            <width>20</width>
            <height>40</height>
        </size>
        </property>
        </spacer>
        </item>
        <item>
        <widget class="QLabel" name="label_2">
        <property name="text">
        <string>Stichwort</string>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <spacer name="verticalSpacer_2">
        <property name="orientation">
        <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
        <size>
            <width>20</width>
            <height>40</height>
        </size>
        </property>
        </spacer>
        </item>
        <item>
        <widget class="QLabel" name="label_3">
        <property name="text">
        <string>Fahrzeuge</string>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <widget class="QPlainTextEdit" name="textVehicleList">
        <property name="maximumSize">
        <size>
            <width>16777215</width>
            <height>120</height>
        </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <spacer name="verticalSpacer_3">
        <property name="orientation">
        <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
        <size>
            <width>20</width>
            <height>40</height>
        </size>
        </property>
        </spacer>
        </item>
        <item>
        <widget class="QLabel" name="label_4">
        <property name="text">
        <string>Freitext</string>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        </widget>
        </item>
        <item>
        <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
        </widget>
        </item>
        <item>
        <spacer name="verticalSpacer_4">
        <property name="orientation">
        <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
        <size>
            <width>20</width>
            <height>40</height>
        </size>
        </property>
        </spacer>
        </item>
        <item>
        <widget class="QPushButton" name="pushButton">
        <property name="text">
        <string>Alarmieren</string>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
        </widget>
        </item>
    </layout>
    </widget>
</item>
<item>
    <spacer name="horizontalSpacer_2">
    <property name="orientation">
    <enum>Qt::Horizontal</enum>
    </property>
    <property name="sizeHint" stdset="0">
    <size>
        <width>20</width>
        <height>40</height>
    </size>
    </property>
    </spacer>
    </item>
</layout>
    </widget>
</widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
